docs/application/web/tutorials/sec-privileges.md

```diff
@@ -39,10 +41,10 @@ The following tables list the API privileges, which you must declare when using
 | `http://tizen.org/privilege/appmanager.kill` | partner |  | The application can close other applications. |
 | `http://tizen.org/privilege/appmanager.launch` | public |  | The application can open other applications. |
 | `http://tizen.org/privilege/bluetooth` | public |  | The application can perform unrestricted actions using Bluetooth, such as scanning for and connecting to other devices. |
-| `http://tizen.org/privilege/bluetooth.admin` | public |  | The application can change Bluetooth settings, such as turning Bluetooth on or off, setting the device name, and enabling or disabling AV remote control. |
-| `http://tizen.org/privilege/bluetooth.gap` | public |  | The application can use the Bluetooth Generic Access Profile (GAP). As an example, it can scan and pair with devices. |
-| `http://tizen.org/privilege/bluetooth.health` | public |  | The application can use the Bluetooth Health Device Profile (HDP). As an example, it can send health information. |
-| `http://tizen.org/privilege/bluetooth.spp` | public |  | The application can use the Bluetooth Serial Port Profile (SPP). As an example, it can send serial data. |
+| `http://tizen.org/privilege/bluetooth.admin` | public |  | The application can change Bluetooth settings, such as turning Bluetooth on or off, setting the device name, and enabling or disabling AV remote control. Deprecated since 3.0. |
```

Comment:
Question : We don't have this deprecated information on "old-versioned-sec-privileges.md", even these privileges are deprecated since 3.0. The old-versioned-sec-privileges.md page indicates it is listing the information before 4.0.

Don't we need to add this information to old-versioned-sec-privileges.md, too?

Same to every deprecated privileges before 4.0, except "http://tizen.org/privilege/websetting".

Reply:
updated deprecated information for web old-versioned-sec-privileges.md and modified deprecation phrase of both old-versioned-sec-privileges.md like sec-privileges.md(removed <br> and bold tag).